sfxclient/httpsink_test.go

```diff
@@ -227,3 +229,155 @@ func ExampleHTTPDatapointSink() {
 		panic(err)
 	}
 }
+
+func TestHTTPEventSink(t *testing.T) {
```

Comment:
Couldn't automatically clear this comment, but the "points" have been changed to "events".